### ISSUE TITLE
[fr] Mistranslation of "Change Tab Display Size" in french, and mentions of tabs in this particular context

### ISSUE THREAD

**Issue by Lostmindy** [microsoft/vscode-loc#1737]
Hello !

The text *change tab display size* is currently translated as "Modifier la taille de l'onglet" and should be instead translated as "_Modifier la taille des  tabulations_". This text is found when you click at the "spaces" info at the bottom of the editor to change the related options.

Mention of tabs in this specific context should be translated as "_tabulations_" and not "_onglets_". "_onglets_" in french are referring to the little tabs at the top of the window to change what file you're viewing. "_tabulations_" are the typographic spaces in code. Both use the same word "tabs" in English.

So to sum it up

| Original English | Wrong translation | Correct translation | 
| --- | --- | --- |
| Change Tab Display Size | Modifier la taille de l'onglet | Modifier la taille des tabulations |
|   Current Tab Size    | Taille actuelle de l'onglet | Taille actuelle des tabulations |
|   Default Tab Size   | Taille d'onglet par défaut | Taille par défaut des tabulations |
| Configured Tab Size | - | Taille des tabulations configurée |

(Yeah that last one is already correctly translated)

Thanks in advance for fixing it!

**Comment by csigs** [comment]
Thank you for your feedback. We will review the issue (ICX ADO Tracking Reference: 1012352).

**Comment by csigs** [comment]
The issue has been fixed in localization platform that the fix should be exported to production repo and will take effect in next release! (Reference id: 1012352)
